src/pages/logos.tsx

```diff
@@ -4,26 +4,77 @@ import { addImageDetails } from "@/shared/addImageDetails";
 import { prisma } from "@/server/db";
 import { Package } from "@prisma/client";
 
+function OffersList({ offers }: { offers: (number[] | boolean)[] }) {
+  const jobTypeNames = [
+    "Summer Job",
+    "Internship",
+    "Part-time Job",
+    "Master Thesis",
+    "Full-time Job",
+    "Trainee Program",
```

Comment:
Use locale for these (translation should already be available) to make these Swedish and English respectively.

Reply:
Fixed now